jobserver/admin.py

```diff
@@ -1,9 +1,22 @@
+import django.contrib.auth.admin  # noqa: F401
+import social_django.admin  # noqa: F401
 from django import forms
 from django.contrib import admin
+from django.contrib.auth.models import Group
+from social_django.models import Association, Nonce, UserSocialAuth
 
 from .models import Org, Project
 
 
+# Remove apps we don't want in the admin
+# Their admin.py files are explicitly imported above so we can unregister them
```

Comment:
Do you mean _implicitly_?

Reply:
Nope, the first two lines of the file import the admin modules which registers their ModelAdmins, so we can deregister them here.

Reply:
So "explicitly imported above", to me, would be `from Foo import admin` - which isn't what happens - do you see what I mean? I didn't get much sleep last night so might be making no sense!

Reply:
I think so… but!

`from Foo import admin` and `import Foo.admin` both import `Foo/admin.py` and are functionally equivalent to trigger the import-time side effects this block of code is undoing (e.g. [the registration of a ModelAdmin](https://github.com/django/django/blob/3.1/django/contrib/auth/admin.py#L25)).  I'd say it's explict because I have a line to import them, whereas if the `jobserver` app was listed in `INSTALLED_APPS` after both `django.contrib.admin` and `social_django` when we wouldn't need the imports.

Reply:
Things happening behind the scenes **as a side effect** of the thing that you actually wrote is the *opposite* of explicit in my book! 

I think this is clearer:

```
# Their admin.py files are imported as a side effect of importing their models, above, so we have to unregister them
```

Anyway /shrug :rofl: not important!